edited db connection and set up data generation script

kelp-chip opened the pull request:
Changed database connection details, successfully generating and saving 10M to a csv file.

kelp-chip committed:
Changed database connection details, successfully generating and saving 10M to a csv file

paulv317 commented on .gitignore:
```diff
@@ -1,6 +1,6 @@
 client/node_modules
 server/node_modules
-
+*.csv
```
very nice!

paulv317 commented on client/package.json:
```diff
@@ -17,6 +17,7 @@
     "babel-jest": "^26.6.3",
     "babel-loader": "^8.2.2",
     "coveralls": "^3.1.0",
+    "css-loader": "^5.0.1",
```
did she not have the css loader?

paulv317 commented on server/.eslintrc.js:
```diff
@@ -7,8 +7,8 @@ module.exports = {
-        "airbnb",
-        "airbnb/hooks"
+        // "airbnb",
+        // "airbnb/hooks"
```
what in the world is this stuff?

kelp-chip replied:
It seems to have it added a airbnb style correction guide. I wasn't crazy about all the squiggly lines but I may add it back at the very end and clean things up that I didn't do correctly. :S

paulv317 commented on server/package.json:
```diff
@@ -18,6 +18,7 @@
     "compression": "^1.7.4",
     "cors": "^2.8.5",
     "express": "^4.17.1",
+    "faker": "^5.3.1",
```
nice

paulv317 commented on server/sdc_db/fakeData.js:
```diff
@@ -0,0 +1,18 @@
+    var num = Math.floor(Math.random() * (2));
+    return num === 0 ? false : true;
+  }
+}
```
very detailed!

paulv317 commented on server/sdc_db/seed.js:
```diff
@@ -0,0 +1,26 @@
+  }
+});
+
+for (let i = 0; i < 100; i++) {
```
100!!?!? What happened to 10 million!?

kelp-chip replied:
Haha! For testing purposes I changed this back to 100 to save time. I plan to compare csv and json creation times!

paulv317 commented on server/sdc_db/seed.js:
```diff
@@ -0,0 +1,26 @@
+  var d = `${hostId},${faker.address.city()},${faker.address.country()},${faker.address.state()},${faker.address.stateAbbr()},${faker.address.longitude()},${faker.address.latitude()},great place to stay,${fakeData.getRandomSmallNumber(10)},${fakeData.getRandomSmallNumber(5)},${fakeData.getRandomSmallNumber(10)},${fakeData.getRandomSmallNumber(5)},${fakeData.getPrice()},${fakeData.getRandomSmallFl ...
+  outputFile.write(d, (err) => {
+    if (err) {
+      console.log(err.message);
```
Oh this is a clean use of those temperate literals(I think that is what they are called)!

paulv317 commented on server/server.js:
```diff
@@ -9,18 +7,18 @@ const db = require('./database');
+const PORT = 3001;
 
 app.get('/lodge', cors(), (req, res) => {
   console.log(`req.query.id ${req.query.id}`);
```
Great job! Keep up the good work!

kelp-chip replied:
Thanks Paul!!!

paulv317 reviewed (commented):
Nice job Lexy!

kelp-chip committed:
Ready to deploy:
json generation script working,
seeding script working,
mongo db now connected to server
routes updated
added post route with limited functionality

kelp-chip committed:
Added files that that didn't make the last commit

kelp-chip committed:
Create generated_data

kelp-chip committed:
Delete generated_data

kelp-chip committed:
Create index.html

kelp-chip committed:
Delete index.html

kelp-chip committed:
Removed new relic

kelp-chip committed:
Connected to remote db
